rpmlint/descriptions/PythonCheck.toml

```diff
@@ -0,0 +1,25 @@
+python-tests-in-package="""
+test/ or tests/ directory in package directory. Tests should not be included in %{python_sitelib}/<pkgname>."""
+python-doc-in-package="""
+doc/ or docs/ directory in package directory. Documentation should go into %{docdir}, not %{python_sitelib}/<pkgname>"""
+python-src-in-package="""
+src/ directory included in %{python_sitelib}/<pkgname>. This should be in a separate -src package.
+"""
```

Comment:
What guidelines are enforcing this? There doesn't seem to be anything in Fedora or openSUSE for it.

Reply:
Guess what happens if two separate packages install site-packages/src or site-packages/test....

Reply:
That's not what this check is about? This seems to be blocking `%python_sitelib/<pkgname>/{src,tests,docs}`?

Reply:
It ain't blocking but warning about those, as it is in most cases just wasted disk space esp. on containers.

Reply:
that is indeed weird.